pandas/tests/test_window.py

```diff
@@ -483,6 +496,35 @@ def test_numpy_compat(self):
             tm.assert_raises_regex(UnsupportedFunctionCall, msg,
                                    getattr(e, func), dtype=np.float64)
 
+    # TODO: GH 16425: Add '1s' datetime expander when GH 16425 is resolved
+    @pytest.mark.parametrize('expander', [1])
+    def tests_empty_df_expanding(self, expander):
+        # Verifies that datetime and integer expanding windows can be applied to empty DataFrames
+        # GH 15819
+        expected = DataFrame()
+        result = DataFrame().expanding(expander).sum()
+        tm.assert_frame_equal(result, expected)
+
+        # Verifies that datetime and integer expanding windows can be applied to empty DataFrames with datetime index
+        expected = DataFrame(index=pd.DatetimeIndex([]))
+        result = DataFrame(index=pd.DatetimeIndex([])).expanding(expander).sum()
+        tm.assert_frame_equal(result, expected)
+
+    # TODO: GH 16425: Remove this test and add '1s' to roller parameter of test_empty_df_expanding() parameter
+    # when GH 16425 is resolved
+    @pytest.mark.xfail(reason="Open issue GH 16425")
```

Comment:
make a referece to what this issue is fixing as well (like I showed above), this is a printed warning when running the tests e.g.

``pytest ..... -r x`` will show them